docs/source/guide/calibrators.md

```diff
@@ -57,6 +57,22 @@ print(cal.get_cost())
 cal.run()
 ```
 
+## Verbose Log Output
+
+The `log` parameter can be passed in to the `run` method with either the value of `flat` or `cartesian`. By using the `log` parameter, detailed information about each experiment is shown. 
+
+The log output can be generated from the results of the calibrator by calling either [`log_results_flat()`](https://mitiq.readthedocs.io/en/stable/apidoc.html#mitiq.calibration.calibrator.ExperimentResults.log_results_flat) or [`log_results_cartesian()`](https://mitiq.readthedocs.io/en/stable/apidoc.html#mitiq.calibration.calibrator.ExperimentResults.log_results_cartesian). 
```

Comment:
1. Let's rewrite this sentence to make it clear you can you use `run` to log at runtime, or either of this functions to log _after_ experiments are run.
2. Let's use the ``{func}`.function_name` `` pattern to generate links. [Here](https://github.com/unitaryfund/mitiq/blob/33151844f54e3bee1f41ffd406c3783022bc9f6a/docs/source/guide/zne-1-intro.md?plain=1#L99) is an example where we use this.

Reply:
Let me know what you think about the re-worded description.

Also, for the {func} I tried just using {func}`.function_name` and at least locally it didn't link appropriately. I found that you can do {func}`module_name.function_name`. I tried some shortcuts with that as well, but was unable to get it to work locally without listing everything as can be seen in the latest commit. 

https://github.com/unitaryfund/mitiq/pull/2568/commits/335b5048443cdb22155712ff0ad350a80c007531

(Side note: It looks like even in the ZNE file you pointed to there is a big mix of full URLs and {func}.)

Reply:
Ah since this is a class method, you should be able to use `{meth}` instead of `{func}`! I think something like ``{meth}`ExperimentResults.func_name` ``.